status_go.nim

```diff
@@ -11,166 +12,270 @@ import ./status_go/impl as go_shim
 export SignalCallback
 
 proc hashMessage*(message: string): string =
-  $go_shim.hashMessage(message.cstring)
+  var funcOut:cstring = go_shim.hashMessage(message.cstring)
```

Comment:
In impl.nim, the return type is `csrting` so is it necessary to explicitly type `var funcOut` here? Same question for all the other uses of `funcOut`.